doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+cd ~/fcm_tutorial
+svnadmin create repository
+</pre>
+                <br/>
+                This creates a repository called <samp>repository</samp> using the 
+                <code>svnadmin</code> tool.
+                <pre>
+fcm mkdir file:///$HOME/fcm_tutorial/repository/fundamentals -m 'New project fundamentals'
+</pre>
+                <br/>
+                using the <code>-m option</code> means that we do not have to add the comment 
+                to the editor which is involved by the command for the commit message.
+                <br/>
+                We can see from the command that we can use <code>mkdir</code> to make the 
+                directory like we would in Linux. Many of the commands that we can use are 
+                the same as we would use in Linux to help make it easier for the user.
+                <br/>
+                The returned message is:
+                <pre>
+<font color="blue">
+Committed revision 1.
+</font>
+</pre>
+                <br/>
+                <pre>
+fcm mkdir file:///$HOME/fcm_tutorial/repository/fundamentals/trunk -m 'New folder fundamentals/trunk'
+fcm mkdir file:///$HOME/fcm_tutorial/repository/fundamentals/branches -m 'New folder fundamentals/branches'
+</pre>
+                <br/>
+                The returned messages are:
+                <pre>
+<font color="blue">
+Committed revision 2.
+
+Committed revision 3.
+</font>
+</pre>
+                respectively.
+                <br/>
+                <pre>
+fcm ls file:///$HOME/fcm_tutorial/repository/fundamentals
+</pre>
+                This lists the changes to the repository and returns:
+                <pre>
+<font color="blue">
+branches/
+trunk/
+</font>
+</pre>
+                Now to make our working copy, this uses Linux <code>mkdir -p</code> to create 
+                the directory and the path to it if it does not already exist:
+                <pre>
+mkdir -p ~/my_fcm_tutorial/fundamentals
+</pre>
+                Checkout the working copy using the Fcm command <code>fcm co</code>:
+                <pre>
+fcm co file:///$HOME/fcm_tutorial/repository/fundamentals/trunk ~/my_fcm_tutorial/fundamentals/trunk
+</pre>
+                Returns:
+                <pre>
+<font color="blue">
+Checked out revision 3.
+</font>
+</pre>
+                <br/>
+                We will be using the tutorials repository for the Essential tutorials.
+                This is an example of how we would set-up our own repository. 
+            </div>
+        </ul>
+        <h1><a id="S6">Setting-up the tutorial repositories</a></h1>
+        <ul>
+            <div class="well">
+                Please run the following <a href="./repos/tutorial_repo_setup">tutorial repositories script</a> by running the following command in a terminal 
+                <code>~/kerryday/fcm_tutorial_docs/bootstrap/docs/repos/tutorial_repo_setup</code> this will set-up the repositories that you will need for all of the tutorials.
```

Comment:
I guess this path is going to be replaced to have a central script that can just be invoked as something like `fcm tutorial-repo-setup`?